### DISCUSSION TITLE
Button is not enabling after creating an new list

### DISCUSSION THREAD

**Discussion by MukeshKumarBagaria** [community/community#58026] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

In video i explained about it in detail
https://github.com/community/community/assets/88270096/fa7f4832-d726-4eaa-ac61-af764c86b57c

The button is just in the disabled state, it can give us a response as a message of being created a new list or redirect us to a new list

**Comment by ghost** [comment]
Yes I think also if he add more representative images it will be good also

**Comment by Sammyreal1** [comment]
Approved

**Comment by Jamim** [comment]
This discussion duplicates #50880, just as a dozen of other reports.
